.azure-pipelines/common-templates/download-openapi-docs.yml

```diff
@@ -8,11 +8,14 @@ parameters:
   - name: BaseBranch
     type: string
     default: "dev"
+  - name: BuildAgent
+    default: MsGraphBuildAgentsLinux
```

Comment:
Can the SDK be built on Linux?

Reply:
It cant because of the net461 Authentication Tests. I reverted the change to use the new updated agent.